custom_components/bonaire_myclimate/BonairePyClimate/hub.py

```diff
@@ -49,7 +47,7 @@ def __init__(self, hass, local_ip):
         self.hvac_modes = None
         self.preset_mode = None
         self.preset_modes = None
-        self.supported_features = 0
```

Comment:
Thanks @boc-the-git. I'm hoping this will fix my HomeKit integration, which tries to simultaneously turn on the unit and set the temperature. My guess is the on/off backward compatibility is confusing HomeKit bridge, so it is sending the wrong sequence of actions.

Reply:
On further testing, the Bonaire unit will not respond to changing the preset or temparture when the unit is off.

The dashboard GUI uses the supported features flags to enable the GUI controls, but they dont work - sending the commands to the Bonaire unit will return a response that ignores the original action.

So I would recommend changing this line to:
`self.supported_features = ClimateEntityFeature.TURN_OFF`

Reply:
Just to confirm @MoFtZ, your conclusion here is that you do like the change I'd implemented and it isn't causing you a (new) problem?

Notwithstanding you've got a PR to add additional functionality.

Reply:
Correct, the PR is fine as-is. Thanks @boc-the-git